### PR TITLE
Add cairolike features to test collector

### PR DESCRIPTION
**Pull request by piotmag769**
Related to https://github.com/foundry-rs/starknet-foundry/issues/2025

### CODE REVIEW

File: extensions/scarb-snforge-test-collector/Cargo.toml [hunk 1]
```diff
@@ -32,6 +32,7 @@ itertools.workspace = true
 num-bigint.workspace = true
 num-traits.workspace = true
 rayon.workspace = true
+scarb = { path = "../../scarb" }
```
Comment (maciektr) [inline extensions/scarb-snforge-test-collector/Cargo.toml:35]: Remove

File: extensions/scarb-cairo-test/src/main.rs [hunk 2]
```diff
@@ -8,7 +8,7 @@ use clap::Parser;
 use scarb_metadata::{
     Metadata, MetadataCommand, PackageId, PackageMetadata, ScarbCommand, TargetMetadata,
 };
-use scarb_ui::args::{FeaturesSpec, PackagesFilter};
+use scarb_ui::args::PackagesFilter;
```
Comment (maciektr) [inline extensions/scarb-cairo-test/src/main.rs:11]: Rebase.

File: extensions/scarb-snforge-test-collector/src/main.rs [hunk 3]
```diff
@@ -22,12 +23,19 @@ mod metadata;
 struct Args {
     #[command(flatten)]
     packages_filter: PackagesFilter,
+
+    #[command(flatten)]
+    pub features: FeaturesSpec,
```
Comment (maciektr) [inline extensions/scarb-snforge-test-collector/src/main.rs:28]: When I think about this now, this env should be set by Foundry and just inherited downstream, no need for cli parser.

Comment (piotmag769) [reply extensions/scarb-snforge-test-collector/src/main.rs:28]: Hmmm okay I guess

**Comment by maciektr** [comment]
Also, *cairolike, we didn't copy the concept for nothing.

**Review by maciektr — APPROVED** [review]

**Pull request merged**